coalib/results/Diff.py

```diff
@@ -20,11 +20,14 @@ def __init__(self, file_list=None,
         Creates an empty diff for the given file.
 
         :param file_list: The original (unmodified) file as a list of its
-                          lines.
+                          lines. It has to be provided if create is not set,
+                          else an error will be raised.
         :param rename:    False or str containing new name of file.
         :param delete:    True if file is set to be deleted.
         :param create:    False or str containing the file name of the file to
-                          be created.
+                          be created. If this propoerty is set and file_list
+                          is passed, then the lines in the ``file_list``
+                          are added(``Diff.add_lines``) to the diff.
```

Comment:
1. Then say `(via Diff.add_lines)` to make that clear :) 2. implementation detail, would omit that. :)